debugger/pythonExtension/gdb_sgx_plugin.py

```diff
@@ -78,7 +78,7 @@ def is_valid(self):
 
 
 # This constant definition must align with the OE enclave layout.
-TD_OFFSET_FROM_TCS =  0X4000
+TD_OFFSET_FROM_TCS =  0X5000
```

Comment:
I had a discussion with Alvin about this. With the work that we are doing with the debugger teams, the reliance of TD_OFFSET_FROM_TCS will go away very soon. So, there is no value in making this robust as part of this PR, since the work we are doing with the debugger teams will supersede this change.

Reply:
Oh I see this comment! Okay this gives total context into the changes in your other PR. Cool 😎